datasets/doc/source/how-to-use-with-torch.rst

```diff
@@ -0,0 +1,51 @@
+Use with PyTorch
```

Comment:
In the framework docs, we spell out the full framework name also in the filename (`how-to-use-with-pytorch.rst`)